Demo/Demo/SwiftUIComponents/CardPaymentViews/CardPaymentOrderCompletionView.swift

```diff
@@ -6,21 +6,32 @@ struct CardPaymentOrderCompletionView: View {
     @ObservedObject var cardPaymentViewModel: CardPaymentViewModel
 
     var body: some View {
-        VStack {
-            CardApprovalResultView(cardPaymentViewModel: cardPaymentViewModel)
-            if cardPaymentViewModel.state.approveResult != nil {
-                CardOrderActionButton(
-                    intent: cardPaymentViewModel.state.intent,
-                    orderID: orderID,
-                    selectedMerchantIntegration: DemoSettings.merchantIntegration,
-                    cardPaymentViewModel: cardPaymentViewModel
-                )
-            }
+        ScrollView {
+            ScrollViewReader { scrollView in
+                VStack {
+                    CardApprovalResultView(cardPaymentViewModel: cardPaymentViewModel)
+                    if cardPaymentViewModel.state.approveResult != nil {
+                        CardOrderActionButton(
+                            intent: cardPaymentViewModel.state.intent,
+                            orderID: orderID,
+                            selectedMerchantIntegration: DemoSettings.merchantIntegration,
+                            cardPaymentViewModel: cardPaymentViewModel
+                        )
+                    }
 
-            if cardPaymentViewModel.state.authorizedOrder != nil || cardPaymentViewModel.state.capturedOrder != nil {
-                CardOrderCompletionResultView(cardPaymentViewModel: cardPaymentViewModel)
+                    if cardPaymentViewModel.state.authorizedOrder != nil || cardPaymentViewModel.state.capturedOrder != nil {
```

Comment:
It looks like we use `cardPaymentViewModel.state` across this view quite a bit. Should we consider extracting it out into a constant?

Reply:
Addressed here: 8a9996c0037e4b0c6086aff0de882bfb42881fe1